specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1267 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Cancel a run with URI": {
+            "$ref": "./examples/cancelRunWithUri.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IFormFile": {
+      "type": "object",
+      "properties": {
+        "contentType": {
+          "type": "string",
+          "readOnly": true
+        },
+        "contentDisposition": {
+          "type": "string",
+          "readOnly": true
+        },
+        "headers": {
+          "type": "object",
+          "additionalProperties": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "readOnly": true
+        },
+        "length": {
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "fileName": {
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "FileStreamResult": {
+      "type": "object",
+      "properties": {
+        "fileStream": {
+          "$ref": "#/definitions/Stream"
+        },
+        "contentType": {
+          "type": "string",
+          "readOnly": true
+        },
+        "fileDownloadName": {
+          "type": "string"
+        },
+        "lastModified": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "entityTag": {
+          "$ref": "#/definitions/EntityTagHeaderValue"
+        }
+      }
+    },
+    "Stream": {
+      "type": "object",
+      "properties": {
+        "canRead": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "canSeek": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "canTimeout": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "canWrite": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "length": {
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true
+        },
+        "position": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "readTimeout": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "writeTimeout": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "EntityTagHeaderValue": {
+      "type": "object",
+      "properties": {
+        "tag": {
+          "$ref": "#/definitions/StringSegment",
+          "readOnly": true
+        },
+        "isWeak": {
+          "type": "boolean",
+          "readOnly": true
+        }
+      }
+    },
+    "StringSegment": {
```

Comment:
What does this type do? It seems like it was intended to be a string, but it's some kind of buffer type.

Reply:
This definition needs to be removed. fixed